src/screens/PaymentDetailsScreen.tsx

```
@@ -5,25 +1,155 @@
+        <div key={member.name} className="field is-horizontal">
+          <div className="field-label is-normal is-expanded">
+            <label className="label is-size-5">{member.name}</label>
+          </div>
+          <div className="field-body is-narrow">
+            <div className="field has-addons">
+              <p className="control">
+                <span className="button is-static is-small">$</span>
+              </p>
+              <p className="control" style={{width: "120px"}}>
+                <input
+                  className={`input is-small ${formError && formError.includes('amount') ? 'is-danger' : ''}`}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)